core/unix/loader.c

```diff
@@ -2195,4 +2195,15 @@ privload_early_inject(void **sp, byte *old_libdr_base, size_t old_libdr_size)
 #    endif /* !defined(STANDALONE_UNIT_TEST) && !defined(STATIC_LIBRARY) */
 #else
 /* XXX i#1285: implement MacOS private loader */
+void
+relocate_dynamorio(byte *dr_map, size_t dr_size, byte *sp)
+{
+    abort();
```

Comment:
Please include TODO comments with issue numbers for anything not implemented, and use ASSERT_NOT_IMPLEMENTED (maybe followed by os_terminate if really worried about release build), not libc routines.  Ditto below.